testsuite/openshift/metrics/prometheus.py

```diff
@@ -73,7 +74,7 @@ def wait_for_scrape(self, target_service: str, metrics_path: str = "/metrics"):
         """Wait before next metrics scrape on service is finished"""
         call_time = datetime.utcnow()
 
-        @backoff.on_predicate(backoff.constant, interval=10, jitter=None, max_tries=10)
+        @backoff.on_predicate(backoff.constant, interval=10, jitter=None, max_tries=20)
```

Comment:
Do we really need this? and if so, do we know why it needs to be that big, what is causing it to be that horribly slow?